src/dali/plugin/pair_converter/ccxt_converter.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2022 macanudo527
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import re
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, NamedTuple, Optional, Set, Union
+
+from ccxt import binance
+from rp2.logger import create_logger
+from rp2.rp2_decimal import RP2Decimal
+
+from dali.abstract_pair_converter_plugin import AbstractPairConverterPlugin
+from dali.historical_bar import HistoricalBar
+
+# Native format keywords
+_ID: str = "id"
+
+# Time in ms
+_MINUTE: str = "1m"
+_FIVEMINUTE: str = "5m"
+_FIFTEENMINUTE: str = "15m"
+_ONEHOUR: str = "1h"
+_FOURHOUR: str = "4h"
+_ONEDAY: str = "1d"
+_TIMEGRANULARITY: List[str] = [_MINUTE, _FIVEMINUTE, _FIFTEENMINUTE, _ONEHOUR, _FOURHOUR, _ONEDAY]
+_TIMEGRANULARITY_IN_S: List[int] = [60, 300, 900, 3600, 14400, 86400]

```

Comment:
According to the [PEP8 style guide](https://peps.python.org/pep-0008/#constants) constants should be capital case with words separated by underscore: _FIVE_MINUTE, ONE_HOUR, etc.